U/UMAP/Compat.toml

```diff
@@ -1,11 +1,17 @@
 [0]
 Arpack = "0.3-0"
-Distances = "0.7-0"
-LsqFit = "0.6-0"
 julia = "1"
 
 ["0-0.1.0"]
 NearestNeighborDescent = "0.1.4-0"
 
+["0-0.1.2"]
+Distances = "0.7-0"
+LsqFit = "0.6-0"
+
 ["0.1.1-0"]
 NearestNeighborDescent = "0.2-0"
+
+["0.1.3-0"]
+Distances = "0.8-0"
+LsqFit = "0.6"
```

Comment:
Why is this not `0.6-0` @StefanKarpinski?

Reply:
Because 0.7 and 0.8 versions of LsqFit exist so the incompatibility is not hypothetical.